tests/test_dedisperse.py

```diff
@@ -25,5 +26,5 @@ def test_dedisperse(self):
             expect moved frequencies per sample
         """
         disp_measure = 6
-        results = dedisperse.dedisperse(self.samples, disp_measure)
+        results = dedisperse.dedisperse(np.array(self.samples), None, None, disp_measure)
```

Comment:
None, None isn't really necessary. Can you change this or is there any specific reason why you are doing this?